omnigan/trainer.py

```diff
@@ -1622,9 +1633,15 @@ def resume(self, inference=False):
         # -----  Single Model Loading  -----
         # ----------------------------------
         else:
-            load_path = Path(self.opts.output_path) / Path(
-                "checkpoints/latest_ckpt.pth"
-            )
+            if self.opts.train.resume_by_pth_path:
+                load_path = Path(self.opts.output_path)
+                assert (
+                    os.path.splitext(load_path)[-1] == ".pth"
```

Comment:
it's safer to use pathlib when handling paths.  This would simply be `Path(load_path).suffix == ".pth"`